### ISSUE TITLE
As a user, I want to know that the software is still executing on long running jobs.

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/deep-archive#136]
## 🧑‍🔬 User Persona(s)

Node Operator

## 💪 Motivation
...so that I can be sure the software is still working and is not timing out.

## 📖 Additional Details

When running `pds-deep-archive-registry` for bundle `urn:nasa:pds:mess_rs_derived::3.0`, it takes a long time and it just hangs for a while as it is connecting with the API and parsing through the data. Ideally we would have some sort of heartbeat to indicate the software is still working. Doesn't need to be all that often, but something to show it is still working.

NOTE: The same applies to `pds-deep-archive`

## ⚖️ Acceptance Criteria
**Given** 
**When I perform** 
**Then I expect**

**Comment by rgdeen** [comment]
Upvote.  You can `ls -l` some files in /tmp but that does not provide any indication of how far along it is.  It took 3 days to run InSight (which is not that big an archive) with no indication of where it was.

**Comment by rgdeen** [comment]
Yeah.  I kind of like the idea of moving it to info or even higher (I imagine debug/info spews a lot per entry so every-100 doesn't save you much).  And it might be nice for 100 to be adjustable.  But those are niceties.  Apologies for missing the update.

**Comment by jordanpadams** [comment]
@nutjob4life let's move those to info messages if that works for you?

**Comment by nutjob4life** [comment]
@jordanpadams can do! 👍

**Comment by jordanpadams** [comment]
Skipping I&T. Simple fix that does not need testing.
